File: nathan/Problema0013/problema0013.py
precoFabrica = float(input("Digite o preço de fábrica do veículo: "))
percDistribuidor = float(input("Digite o percentual de lucro do distribuidor: "))
percImposto = float(input("Digite o percentual de impostos: "))

lucroDistribuidor = (precoFabrica * percDistribuidor) / 100
valorImpostos = (precoFabrica * percImposto) / 100
precoFinal = precoFabrica + lucroDistribuidor + valorImpostos

print("Lucro do distribuidor: ", lucroDistribuidor)
print("Valor correspondente aos impostos: ", valorImpostos)
print("Preço final do veículo: ", precoFinal)

File: nathan/Problema0013/problema0013.java
package Problema0013;

import java.util.Scanner;

public class problema0013 {
    static Scanner input = new Scanner(System.in);
   
    public static void main(String[] args) {
        
        System.out.println("Digite o preço de fábrica do veículo: ");
        double precoFabrica = Double.valueOf(input.nextLine());
        System.out.println("Digite o percentual de lucro do distribuidor: ");
        double percDistribuidor = Double.valueOf(input.nextLine());
        System.out.println("Digite o percentual de impostos: ");
        double percImposto = Double.valueOf(input.nextLine());
        
        double lucroDistribuidor = (precoFabrica * percDistribuidor) / 100;
        double valorImpostos = (precoFabrica * percImposto) / 100;
        double precoFinal = precoFabrica + lucroDistribuidor + valorImpostos;
        
        System.out.println("Lucro do distribuidor: " + lucroDistribuidor);
        System.out.println("Valor correspondente aos impostos: " + valorImpostos);
        System.out.println("Preço final do veículo: " + precoFinal);
    }
}
